doc/data_checklist.md

```diff
@@ -0,0 +1,68 @@
+## Working With Large SQL Data Dumps
+
+Working with large SQL dump files can be a high risk endeavor;
+the size of some files can cause a small mistake early on to 
+trash hours of work dumping, zipping, uploading and importing
+the file. To ensure we are measuring twice and cutting once,
+copy and paste this checklist into issues that involve loading
+new SQL data. For example, issues like #1531, #2532, #1408.
+
+## Checklist
+
+- [ ] Load or create the data/updates on your local Postgres instance
+- [ ] Verify the table you are about to dump
+   - Inspect the table and ensure all desired columns and indices are present.
+
+```bash
+./scripts/manage.sh dbshell
+mmw>  \d the_table_name
+```
+**Things to consider:**
+   - How many rows are present?
+   - Are there columns that should have unique values, but don't?
+   - What columns will we primarily operate on? Does every row contain a value for these columns? Should they?
+   - Will we be performing joins, spatial or otherwise, with other tables? How performant are these joins? Would they benefit from a new index?
+
+- [ ] `pg_dump` it!
+   - If you're exporting a table, the command may look like this:
+```bash
+$ vagrant ssh services
+$ pg_dump -c --if-exists -O
+        -t the_table_name \
+        -h localhost -U mmw mmw \
+    > /vagrant/thedump.sql
+```
+
+- [ ] Sanity check the `pg_dump` output
+```bash
+# The first 40-60 lines will likely contain 
+# DROP and CREATE_TABLE statements you should verify.
+head -n 45 thedump.sql
+
+# The end of the file will likely create
+# any required indices.
+# It should also contain a comment stating the pg_dump
+# is complete.
+tail thedump.sql
+```
+- [ ] Gzip and upload to data bucket
+```bash
+./scripts/data/upload_sql_data.sh [-p MMW_STG_AWS_PROFILE] PATH_TO_FILE [DST_FILE_NAME]
+# eg
+./scripts/data/upload_sql_data.sh -p mmw-stg ./data/new_nhdflowlines_with_slope.sql nhdflowline
+# Will create s3://data.mmw.azavea.com/nhdflowline_1513899196.sql.gz
```

Comment:
The base filename on line 52 differs from the filename in like 53. Is that intentional?

Reply:
It is. The final parameter of the script is the desired output file name, in this case `nhdflowline`. The script uploads the gzipped file as `NAME_<timestamp>`, so its `nhdflowline_1513899196.sql.gz`. If you leave off the output name parameter it will use whatever name the original file had.